src/public/secure/admin.ejs

```diff
@@ -104,6 +104,22 @@
             </div>
         </div>
     </div>
+
+    <div class="modal fade" id="fileDetailsModel" tabindex="-1" aria-labelledby="fileDetailsModelLabel"
+         aria-hidden="true">
+        <div class="modal-dialog" style="max-width:50% !important;">
```

Comment:
i don't think we should be adding width, boostrap allows you to size them https://getbootstrap.com/docs/5.3/components/modal/#optional-sizes

Reply:
I dislike this, adding the width - will look at the optional sizes

Reply:
Implemented